src/components/SendTransactionFlow.tsx

```diff
@@ -0,0 +1,350 @@
+import React, { useState } from "react";
+// import { Action } from "@reduxjs/toolkit";
+import StellarSdk from "stellar-sdk";
+import styled from "styled-components";
+import BigNumber from "bignumber.js";
+import { sendTxAction } from "ducks/sendTransaction";
+import { useDispatch } from "react-redux";
+import { useRedux } from "hooks/useRedux";
+import { ActionStatus } from "ducks/account";
+import { loadPrivateKey } from "helpers/keyManager";
+
+const El = styled.div``;
+
+const TempButtonEl = styled.button`
+  margin-bottom: 20px;
+`;
+
+const TempInputEl = styled.input`
+  margin-bottom: 20px;
+  min-width: 300px;
+`;
+
+const TempSelectInputEl = styled.select`
+  margin-bottom: 20px;
+  min-width: 300px;
+`;
+
+const TempAnchorEl = styled.a`
+  display: block;
+  margin-bottom: 20px;
+  cursor: pointer;
+  text-decoration: underline;
+`;
+
+// CREATE -> CONFIRM -> SUCCESS || ERROR
+const sendTxFlowEnum = {
+  CREATE: 0,
+  CONFIRM: 1,
+  SUCCESS: 2,
+  ERROR: 3,
+};
+
+interface FormData {
+  toAccountId: string;
+  amount: BigNumber;
+  fee: string;
+  memoType: string;
+  memoContent: string;
```

Comment:
Are `memoType` and `memoContent` required in this case? We can use types `MemoType` /`MemoValue` from `stellar-sdk` for them.